apis/otpapi/otpapi.yaml

```diff
@@ -0,0 +1,210 @@
+              -  Email is invalid.
+              -  Please provide a valid phone number.
+              -  Invalid type. Valid values are: email, phone.'
+          schema:  
+            $ref: '#/definitions/CommonErrorResponse'
+        '429':
+          description: 'TOO_MANY_REQUESTS. The requested operation failed. Possible reasons for failure:
+              - Your per hour/day rate limit has exceeded. You can retry after some time.'
+          schema:  
+            $ref: '#/definitions/CommonErrorResponse'    
+        '500':
+          description: 'INTERNAL SERVER ERROR. The requested operation failed due to a server error.'
+  /verify:
+    post:
+      description: >-
+       This API is associated with verifying otp send to user on Sunbird platform.
+          - The **_/otp/v1/verify_** endpoint executes the "Verify OTP" request based on parameters provided as metadata in the request body
+          - Required fields cannot be left null or empty
+      summary: Verify OTP
+      tags: 
+      - OTP APIs
+      operationId: VerifyOTP
+      produces:
+      - application/json
+      parameters:
+      - name: Body
+        in: body
+        required: true
+        description: >-
+            The body refers to the format of the request.
+              - The body contains metadata about the request for verifying OTP.
+        schema:  
+          $ref: '#/definitions/VerifyOTPrequest'
+      - name: Content-Type
+        in: header
+        required: true
+        type: string
+        description: >-
+           Type of the request data. Set as application/json.
+      - name: ts
+        in: header
+        required: true
+        type: string
+        description: 'Timestamp at which given API request is sent.'
+      - name: X-msgid
+        in: header
+        required: true
+        type: string
+        description: 'This ID uniquely identifies a request if the same API is executed multiple times.'
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: 'Specify authorization key (format: Bearer api-key) received from administrator when performing given API request.'
+      responses:
+        '200':
+          description: 'OK. Successful operation. The requested operation was successfully executed.'
+          schema:  
+             $ref: '#/definitions/CommonSuccessResponse'
+        '400':
+          description: 'CLIENT_ERROR. The requested operation failed due to bad request from client. Possible reasons for failure:
+              -  Mandatory parameter is missing.
+              -  Email is invalid.
+              -  Please provide a valid phone number.
+              -  Invalid type. Valid values are: email, phone.
+              -  Invalid OTP.'
+          schema:  
+            $ref: '#/definitions/CommonErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR. The requested operation failed due to a server error.'
```

Comment:
INTERNAL SERVER ERROR, we are tracking the error and cater to it on priority. Try to refresh the page and if the problem persists contact us at info@sunbird.org